version.go

```diff
@@ -0,0 +1,21 @@
+package main
```

Comment:
Why is this in the main package?
The version should be defined in main.go. This file should move to the commands package.